### DISCUSSION TITLE
which diff between query and modifyQueryUsing method in Components\Tab?

### DISCUSSION THREAD

**Discussion by alisalehi1380** [filamentphp/filament#13238] [General]
![image](https://github.com/filamentphp/filament/assets/111766206/69c41e81-251c-4ec3-8fbc-ac4338de6e18)

which diff between these two methods in `Filament\Resources\Components\Tab`?

**Comment by dissto** [comment]
`query()` is just an alias for `modifyQueryUsing()` in this case
